### ISSUE TITLE
how to compute mAp

### ISSUE THREAD

**Issue by Boyi92** [victordibia/handtracking#4]
I want to know how you compute the mAP and let it show in the tensorboard? I'm glad to receive a reply from you,thank you

**Comment by victordibia** [comment]
Hi @Boyi92 

While you are running your training on your data, you can run a concurrent evaluation job that evaluates your trained model on your test set.  If you are using GCP ML Engine to run your train/evaluation jobs, you can follow the [guide here](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_on_cloud.md#running-an-evaluation-job-on-cloud). 

As your evaluation job progresses, you can run a tensorboard instance and point it to the evaluation job directory ... at that point can now view mAP on tensorboard. 

![image](https://user-images.githubusercontent.com/1547007/34925782-809563e8-f979-11e7-91e5-5dd97b8817cd.png)

To learn more about how the evaluation job works, the metrics it provides (mAP), you can look at the code in `[eval.py](https://github.com/tensorflow/models/blob/master/research/object_detection/eval.py)` and `[evaluator.py](https://github.com/tensorflow/models/blob/master/research/object_detection/evaluator.py)` and `[evaluation util](https://github.com/tensorflow/models/blob/master/research/object_detection/utils/object_detection_evaluation.py)` provided in the object detection api repo.


-V.

**Comment by Boyi92** [comment]
I'm glad to recieve your reply, thank you very much.
